app/controllers/graphql_controller.rb

```diff
@@ -0,0 +1,48 @@
+class GraphqlController < ApplicationController
+  # If accessing from outside this domain, nullify the session
+  # This allows for outside API access while preventing CSRF attacks,
+  # but you'll have to authenticate your user separately
+  # protect_from_forgery with: :null_session
+
+  def execute
+    variables = ensure_hash(params[:variables])
+    query = params[:query]
+    operation_name = params[:operationName]
+    context = {
+      # Query context goes here, for example:
+      # current_user: current_user,
+    }
+    result = TreatYoSelfSchema.execute(query, variables: variables, context: context, operation_name: operation_name)
+    render json: result
+  rescue => e
+    raise e unless Rails.env.development?
+    handle_error_in_development e
+  end
```

Comment:
Interesting setup. Is this some boilerplate functionality provided by GraphQL that you modify for your specific installation instance? 

Excited to learn more about this, so maybe a point we can reference in our overview.

Reply:
Yes these are autogenerate files that GraphQL provides when you install it on in your app.